src/Analysers/TTbar_plus_X_analyser.cpp

```diff
@@ -260,8 +276,8 @@ void TTbar_plus_X_analyser::ePlusJetsQcdAnalysis(const EventPtr event) {
 		}
 
 		// MET Uncertainties
-		for ( unsigned int unc_i = 0; unc_i < MET_main->getAllMETUncertainties().size(); ++unc_i ) {
-			METPointer METForUnc_i = MET_main->getMETForUncertainty( unc_i );
+		for ( unsigned int unc_i = 0; unc_i < METNoHF->getAllMETUncertainties().size(); ++unc_i ) {
+			METPointer METForUnc_i = METNoHF->getMETForUncertainty( unc_i );
```

Comment:
So the uncertainties are only valid for METNoHF at the moment. We will probably need this to be a config option later.